Jenkinsfile

```diff
@@ -142,6 +142,9 @@ try {
 					// Grab the commit that triggered the build.
 					checkout scm
 
+					// Address limitations resulting from CVE-2022-24767
+					sh 'git config --global --add safe.directory "$WORKSPACE"'
```

Comment:
Note: This is already present elsewhere in our Jenkins builds, but this is the first time we must introduce this to our _root_ Jenkinsfile with a more modern version of git present. In short, newer versions of git prevent git operations when there is so-called _dubious_ ownership, i.e. when the cloning user/uid differs from the the user/uid that calls subsequent git operations, yielding a fatal error with a message like the following:
> fatal: detected dubious ownership in repository at '/home/jenkins/agent/workspace/_and_Multistage_Pipeline_PR-1672'
> To add an exception for this directory, call:
>
>	git config --global --add safe.directory /home/jenkins/agent/workspace/_and_Multistage_Pipeline_PR-1672

Here, we take advantage of the Jenkins built-in `$WORKSPACE` environment variable, marking it as safe for any subsequent git interaction.